Update sales process docs (#309)
2i2c-org/team-compass: projects/managed-hubs/sales.md

# Sales and invoicing process

This section has information about pursuing and creating new contracts for 2i2c services.

## Location of sales assets and documents

Here are some locations of common documents used in the sales process (other than the repositories described below).

[The Managed Hub Service folder](https://drive.google.com/drive/folders/1QuvUvwFxPAxw-bJ6_zjgwKXPurHC6UfW?usp=sharing)
: Contains all folders and information that cover the Managed Hub Service, including assets for creating new contracts and following leads.

[The leads and partnerships AirTable](https://airtable.com/appDUEjjcSqHfilFD/tblxLUQ3ElIaaQuM8/viwXstuM3K48smZXW)
: An AirTable with the pipeline for potential communities we will serve.
  This table is the source of truth about who we are talking to about serving hub infrastructure, and who we have an active service with.

[The partnerships and leads assets folder](https://drive.google.com/drive/folders/1aMZILBmFSTYBSB9EwyV5wRpcuprM06dJ?usp=sharing)
: Contains all templates, email copy, stationery, etc that is useful in 2i2c sales.

## Who participates in this process?

> Anybody at 2i2c that is interested in growing our service to new communities is welcome to participate in the process below.

Reviewer: @choldgraf how we would manage the private information if everyone is welcome to participate in the process?

Author: by that I mean anybody on the @2i2c-org/tech-team or the @2i2c-org/steering-council

> Anybody at 2i2c that is interested in growing our service to new communities is welcome to participate in the process below.

Author: ```suggestion
Anybody on the core team of 2i2c, or the steering council, is welcome to participate in the process below.
```

Author: @damianavila does this work for you?

Reviewer: Yep, thanks for the clarification!

If you wish to help with sales and growing the service, please send an email to `partnerships@2i2c.org` so you can be added to that list, and be given a quick run-down of the process.

All members of `partnerships@2i2c.org` are invited to participate in the process below.

## Tracking leads with AirTable

We use AirTable to track our leads for prospective communities to serve.
: [**This AirTable**](https://airtable.com/appDUEjjcSqHfilFD/tblxLUQ3ElIaaQuM8/viwXstuM3K48smZXW?blocks=hide) keeps track of all potential communities we may serve. Below is an overview of each step in the process.

Status fields indicate each step of the sales process

> : We use the **status column** to indicate where a community is in the process. There are roughly five statuses that a community can have:

Reviewer: I think you are describing 6 statuses below.

> : We use the **status column** to indicate where a community is in the process. There are roughly five statuses that a community can have:

Author: ```suggestion
: We use the **status column** to indicate where a community is in the process. There are a few statuses that a community can have:
```

  - {guilabel}`Active Lead` is a community that is actively interested in our service.
  - {guilabel}`Inactive Lead` is a potential community that is less-responsive or does not seem interested anymore.
  - {guilabel}`Needs Admin` is a lead that has agreed to a service tier and now needs CS&S to set up administrative infrastructure.
  - {guilabel}`Active Service` is a community for which we have an active service contract.
  - {guilabel}`Inactive Service` is a community that we used to run a service for, but who has since stopped that service.

>   - {guilabel}`Waiting` is a community that is interested in our service, but we are intentionally waiting for an extended period of time to follow-up (for example, if they are waiting to learn if they have received a grant).

Reviewer: I see two dimensions on these statuses... 
One dimension:
Lead > Waiting > Admin > Service
And the other one:
Active > Inactive
Is this how you are envisioning the pipeline?

Author: I think that's right - my thinking is that it's useful if we know whether an organization went from "Lead -> Inactive" or from "Service -> Inactive"

are you suggesting that this would be better-described by 2 metadata fields (one for category, one for active/inactive)?

Reviewer: > are you suggesting that this would be better-described by 2 metadata fields (one for category, one for active/inactive)?

Yep, I think so... otherwise you are "corrupting" a "natural" pipeline with a different level of status...

Author: Would you be OK with us merging as-is, and we can discuss the right pipeline to use for this process in the sustainability team meetings? I think the pipeline should evolve, but this PR at least updates the process so that it reflects our current practice.

Reviewer: That would be OK to me if we follow up later.
